monkey/infection_monkey/pe/actions/__init__.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABCMeta, abstractmethod
+from os.path import dirname, basename, isfile, join, realpath
+import glob
+
+__author__ = 'D3fa1t'
+
+
+class HostPrivExploiter(object):
+    __metaclass__ = ABCMeta
+
+    @abstractmethod
+    def try_priv_esc(self, commad):
+        raise NotImplementedError()
+
+    def send_pe_telemetry(self, result, local_ip):
```

Comment:
I think you'll have to add a new telemetry type in `infection_monkey/telemety` and use it.

Reply:
Ah alright, I didn't see this change.